roles/netbox/tasks/main.yml

```diff
@@ -0,0 +1,112 @@
+---
+- name: Create group
+  ansible.builtin.group:
+    name: "{{ netbox_user_name }}"
+  become: true
+
```

Comment:
Creating a user creates a group per default as well, this step shouldn't be necessary